### ISSUE TITLE
training and valid loss dont converge during sdxl text2image lora training

### ISSUE THREAD

**Issue by DoctorTar** [huggingface/diffusers#5797]
I'm using the train_text_to_image_lora_sdxl.py for lora training on sdxl for specific art style with following args:

accelerate launch $PWD/diffusers/examples/text_to_image/lora_with_valid.py \
  --pretrained_model_name_or_path='../sd_models/stabilityai__stable-diffusion-xl-base-1.0/' \
  --pretrained_vae_model_name_or_path='../sd_models/sdxl_vae_fp16_fix' \
  --train_data_dir='../datasets/traindata' \      _(with 1356 images-texts pair)_
  --caption_column='prompts' \
  --resolution=1024 --center_crop --random_flip \
  --train_batch_size=7 \
  --num_train_epochs=100 \
  --checkpointing_steps=2000 \
  --learning_rate=1e-04 \
  --lr_scheduler="cosine_with_restarts" \
  --lr_warmup_steps=0 \
  --mixed_precision="fp16" \
  --snr_gamma=5 \
  --rank=256 \
  --seed=490 \
  --output_dir="../sd_models/lora_models/$(date +"%Y%m%d.%H%M%S")" \
  --validation_prompt="1girl, brown eyes, closed mouth, earrings, evening, fingerless gloves, fingernails, jewelry, lips" \
  --validation_epochs=20 \

But I found that training loss and validation loss dont converge, which implies the lora model is not learning anything. I've tried variation of args including epoch, lr, rank etc, but it didnt work. Did I do sth wrong or this is that it supposed to be ?
![image](https://github.com/huggingface/diffusers/assets/147801017/df38d541-2456-4e6e-aabe-31ab8beb1b25)
![image](https://github.com/huggingface/diffusers/assets/147801017/876d1be7-b304-4f79-a9de-620b6caa1c6b)

btw, the valid loss part doesnt exist in the original file, I added by myself.

**Comment by yiyixuxu** [comment]
Hi:

can you sample the generations to see if the quality improves? 
loss is not a good indication for diffusion models  

cc @sayakpaul here

**Comment by sayakpaul** [comment]
I am going to echo Yiyi's comments here. 

The loss surface is quite uninformative for diffusion model training at least in these cases. So, it's always better to check if the quality of the generated samples is improving.

**Comment by DoctorTar** [comment]
> I am going to echo Yiyi's comments here.
> 
> The loss surface is quite uninformative for diffusion model training at least in these cases. So, it's always better to check if the quality of the generated samples is improving.

Thanks for ur help, but I still wonder why is happen. We know the network is optimizing the parameters by backward the loss, why the model would  be imporved if the loss is not descending? (Though i do notice generations sample during step is improving)
